pep-0680.rst

```diff
@@ -14,75 +14,89 @@ Post-History: 11-Jan-2022
+<https://github.com/PyCQA/flake8/issues/234#issuecomment-812800657>`__.
 Given the special place TOML already has in the Python ecosystem, it makes sense
-for this to be an included battery.
+for it to be an included battery.
 
-Finally, TOML as a format is increasingly popular (some reasons for this are
-outlined in PEP 518). Hence this is likely to be a generally useful addition,
-even looking beyond the needs of Python packaging and Python tooling: various
-Python TOML libraries have about 2000 reverse dependencies on PyPI. For
-comparison, ``requests`` has about 28k reverse dependencies.
+Finally, TOML as a format is increasingly popular (for the reasons
+outlined in :pep:`518`), with various Python TOML libraries having about
+2000 reverse dependencies on PyPI (for comparison, ``requests`` has about
+28000 reverse dependencies). Hence, this is likely to be a generally useful
+addition, even looking beyond the needs of Python packaging and related tools.
 
 
 Rationale
 =========
 
 This PEP proposes basing the standard library support for reading TOML on the
-third party library ``tomli``
+third-party library ``tomli``
 (`github.com/hukkin/tomli <https://github.com/hukkin/tomli>`_).
 
-Many projects have recently switched to using ``tomli``, for example, ``pip``,
+Many projects have recently switched to using ``tomli``, such as ``pip``,
 ``build``, ``pytest``, ``mypy``, ``black``, ``flit``, ``coverage``,
-``setuptools-scm``, ``cibuildwheel``.
+``setuptools-scm`` and ``cibuildwheel``.
 
-``tomli`` is actively maintained and well-tested. ``tomli`` is about 800 lines
-of code with 100% test coverage and passes all tests in a test suite `proposed
-as the official TOML compliance test suite
-<https://github.com/toml-lang/compliance/pull/8>`_, as well as `the more
-established BurntSushi/toml-test suite
+``tomli`` is actively maintained and well-tested. It is about 800 lines
+of code with 100% test coverage, and passes all tests in the
+`proposed official TOML compliance test suite
+<https://github.com/toml-lang/compliance/pull/8>`_, as well as
+`the more established BurntSushi/toml-test suite
 <https://github.com/BurntSushi/toml-test>`_.
 
 
 Specification
 =============
 
-A new module ``tomllib`` with the following functions will be added:
+A new module ``tomllib`` will be added to the Python standard library,
+exposing the following public functions:
 
 .. code-block::
 
-   def load(fp: SupportsRead[bytes], /, *, parse_float: Callable[[str], Any] = ...) -> dict[str, Any]: ...
-   def loads(s: str, /, *, parse_float: Callable[[str], Any] = ...) -> dict[str, Any]: ...
-
-``tomllib.load`` deserializes a binary file containing a
-TOML document to a Python dict.
+   def load(
+       fp: SupportsRead[bytes],
+       /,
+       *,
+       parse_float: Callable[[str], Any] = ...,
+    ) -> dict[str, Any]: ...
+
+   def loads(
+       s: str,
+       /,
+       *,
+       parse_float: Callable[[str], Any] = ...,
+       ) -> dict[str, Any]: ...
```

Comment:
```suggestion
   ) -> dict[str, Any]: ...
```

Reply:
Oops, somehow missed that one, thanks. Fixed.